cmd/snap-bootstrap/triggerwatch/evdev.go

```diff
@@ -205,6 +205,22 @@ func (e *evdevKeyboardInputDevice) Close() {
 
 type evdevInput struct{}
 
+func (e *evdevInput) Open(filter triggerEventFilter, node string) (triggerDevice, error) {
+	evdevDevice, err := evdev.Open(node)
+	if err != nil {
+		return nil, err
+	}
+	kc, ok := strToKey[filter.Key]
+	if !ok {
+		return nil, fmt.Errorf("cannot find a key matching the filter %q", filter.Key)
+	}
+	cap := evdev.CapabilityCode{Code: kc, Name: filter.Key}
```

Comment:
is it neccessary to declare cap here? it seems like we are only using kc in line 220 - couldn't we just instead do uint16(kc)? Sorry if it is a stupid question and there is something I'm just not seeing

Reply:
Correct. I only extracted the code needed from FindMatchingDevices. But I think I do not properly do the filtering. I will rework that part.

Reply:
I have changed the code to something that makes a bit more sense.